### PR TITLE
fix(DB/Quest): Post-partum Aggression

### PR DESCRIPTION
**Pull request by locus313**
## CHANGES PROPOSED:
-  update smart script for Hemet Nesingwary and Stampy

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3285
- Credits: el-druida

## TESTS PERFORMED:
- Apply sql and restart server
- .tele sholazar
- .quest add 12607
- complete quest A Mammoth Undertaking
- start quest Post-partum Aggression
- kill Broodmother Slivina and verify you got credit while riding the mammoth

## HOW TO TEST THE CHANGES:

- .tele sholazar
- .quest add 12607
- complete quest A Mammoth Undertaking
- start quest Post-partum Aggression
- kill Broodmother Slivina and verify you got credit while riding the mammoth

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Review by Kitzunu — CHANGES_REQUESTED** [review]
Use single quotes and complete the comments pls :)

File: data/sql/updates/pending_db_world/rev_1598936588616883173.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+(28451,0,0,0,0,0,100,0,0,0,2300,3900,11,51742,64,0,0,0,0,2,0,0,0,0,0,0,0,"Hemet Nesingwary - Combat CMC - Cast 'Arcane Shot'"),
+(28451,0,1,0,5,0,100,0,0,0,0,28467,33,28467,0,0,0,0,0,21,10,0,0,0,0,0,0,"Hemet Nesingwary - On Killed Unit - Quest Credit ''");
+
+UPDATE `creature_template` SET `AIName`="SmartAI" WHERE `entry`=28468;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=28468 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(28468,0,0,0,5,0,100,0,0,0,0,28467,33,28467,0,0,0,0,0,21,10,0,0,0,0,0,0,"Stampy - On Killed Unit - Quest Credit ''");
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1598936588616883173.sql:12]: what tool are you using to generate comments?

Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1598936588616883173.sql:12]: he just took this from https://github.com/TrinityCore/TrinityCore/issues/18079#issuecomment-256207459

**Comment by felwyn** [comment]
Okay working.
WoWScrnShot_090120_172955

File: data/sql/updates/pending_db_world/rev_1598936588616883173.sql (hunk 1 above)
Comment (locus313) [reply data/sql/updates/pending_db_world/rev_1598936588616883173.sql:12]: @FrancescoBorzi its as @Helias said. I was only importing the fix.

**Comment by locus313** [comment]
@Kitzunu its updated now.

**Pull request merged**